test_files/class/class.js

```diff
@@ -151,7 +159,7 @@ class AbstractClassImplementsClass {
 }
 /**
  * @abstract
- * @extends {AbstractClass}
+ * @implements {InexpressibleType}
```

Comment:
Is this change kosher?

Reply:
This is changing

```
  class C implements OtherClass
```

where before we did this weird hack around how implements is vaguely the same as 'extends'.  So the diff is showing that I removed the weird hack.  I think it's better to be predictable than fancy here.